[bug]: Fix unit tests in Jane

### What seems to be the problem? 🤔

I merged #6437 because I was worried it would start falling behind, but it needs work.
It has 67 failing unit tests, which should be fixed. Assining to @SeaZeeZee 

### Design / Plugin / Package 🧐

_No response_

### Additional context

```
  1) Pattern translation:
         - The design should have a translated title:

      AssertionError: expected 4 to be above 4
      + expected - actual


      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:17:38)
      at process.processImmediate (node:internal/timers:476:21)

  2) Pattern translation:
         - The translation of o.bicepEase should correspond to a known option:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:74:48)
      at process.processImmediate (node:internal/timers:476:21)

  3) Pattern translation:
         - The option title for the "bicepsEase" option should be a non-empty string:
     TypeError: Cannot read properties of undefined (reading 't')
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/i18n.mjs:89:43)
      at process.processImmediate (node:internal/timers:476:21)

  4) Draft for humans:
       cisFemale
           - Drafting for size 28:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  5) Draft for humans:
       cisFemale
           - Drafting for size 30:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  6) Draft for humans:
       cisFemale
           - Drafting for size 32:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  7) Draft for humans:
       cisFemale
           - Drafting for size 34:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  8) Draft for humans:
       cisFemale
           - Drafting for size 36:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  9) Draft for humans:
       cisFemale
           - Drafting for size 38:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  10) Draft for humans:
       cisFemale
           - Drafting for size 40:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  11) Draft for humans:
       cisFemale
           - Drafting for size 42:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  12) Draft for humans:
       cisFemale
           - Drafting for size 44:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  13) Draft for humans:
       cisFemale
           - Drafting for size 46:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  14) Draft for humans:
       cisMale
           - Drafting for size 32:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  15) Draft for humans:
       cisMale
           - Drafting for size 34:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  16) Draft for humans:
       cisMale
           - Drafting for size 36:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  17) Draft for humans:
       cisMale
           - Drafting for size 38:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  18) Draft for humans:
       cisMale
           - Drafting for size 40:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  19) Draft for humans:
       cisMale
           - Drafting for size 42:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  20) Draft for humans:
       cisMale
           - Drafting for size 44:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  21) Draft for humans:
       cisMale
           - Drafting for size 46:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  22) Draft for humans:
       cisMale
           - Drafting for size 48:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  23) Draft for humans:
       cisMale
           - Drafting for size 50:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:53:20)
      at process.processImmediate (node:internal/timers:476:21)

  24) Draft for doll:
       cisFemale
           - Drafting at 10%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  25) Draft for doll:
       cisFemale
           - Drafting at 20%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  26) Draft for doll:
       cisFemale
           - Drafting at 30%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  27) Draft for doll:
       cisFemale
           - Drafting at 40%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  28) Draft for doll:
       cisFemale
           - Drafting at 50%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  29) Draft for doll:
       cisFemale
           - Drafting at 60%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  30) Draft for doll:
       cisMale
           - Drafting at 10%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  31) Draft for doll:
       cisMale
           - Drafting at 20%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  32) Draft for doll:
       cisMale
           - Drafting at 30%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  33) Draft for doll:
       cisMale
           - Drafting at 40%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  34) Draft for doll:
       cisMale
           - Drafting at 50%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  35) Draft for doll:
       cisMale
           - Drafting at 60%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  36) Draft for giant:
       cisFemale
           - Drafting at 150%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  37) Draft for giant:
       cisFemale
           - Drafting at 200%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  38) Draft for giant:
       cisFemale
           - Drafting at 250%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  39) Draft for giant:
       cisFemale
           - Drafting at 300%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  40) Draft for giant:
       cisMale
           - Drafting at 150%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  41) Draft for giant:
       cisMale
           - Drafting at 200%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  42) Draft for giant:
       cisMale
           - Drafting at 250%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  43) Draft for giant:
       cisMale
           - Drafting at 300%:

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/drafting.mjs:76:22)
      at process.processImmediate (node:internal/timers:476:21)

  44) Sample measurements:
         Sample chest::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:71:16)
      at process.processImmediate (node:internal/timers:476:21)

  45) Sample measurements:
         Sample hips::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:71:16)
      at process.processImmediate (node:internal/timers:476:21)

  46) Sample measurements:
         Sample hpsToWaistBack::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:71:16)
      at process.processImmediate (node:internal/timers:476:21)

  47) Sample measurements:
         Sample waistToKnee::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:71:16)
      at process.processImmediate (node:internal/timers:476:21)

  48) Sample measurements:
         Sample shoulderToShoulder::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:71:16)
      at process.processImmediate (node:internal/timers:476:21)

  49) Sample measurements:
         Sample neck::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:71:16)
      at process.processImmediate (node:internal/timers:476:21)

  50) Sample measurements:
         Sample hpsToBust::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:71:16)
      at process.processImmediate (node:internal/timers:476:21)

  51) Sample measurements:
         Sample waistToFloor::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:71:16)
      at process.processImmediate (node:internal/timers:476:21)

  52) Sample measurements:
         Sample biceps::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:71:16)
      at process.processImmediate (node:internal/timers:476:21)

  53) Sample measurements:
         Sample shoulderToElbow::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:71:16)
      at process.processImmediate (node:internal/timers:476:21)

  54) Sample options:
         Sample bodyEase::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:94:18)
      at process.processImmediate (node:internal/timers:476:21)

  55) Sample options:
         Sample neckDepthFront::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:94:18)
      at process.processImmediate (node:internal/timers:476:21)

  56) Sample options:
         Sample neckDepthBack::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:94:18)
      at process.processImmediate (node:internal/timers:476:21)

  57) Sample options:
         Sample neckWidth::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:94:18)
      at process.processImmediate (node:internal/timers:476:21)

  58) Sample options:
         Sample shiftLength::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:94:18)
      at process.processImmediate (node:internal/timers:476:21)

  59) Sample options:
         Sample bicepsEase::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:94:18)
      at process.processImmediate (node:internal/timers:476:21)

  60) Sample options:
         Sample sleeveBonus::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:94:18)
      at process.processImmediate (node:internal/timers:476:21)

  61) Sample options:
         Sample gussetSize::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:94:18)
      at process.processImmediate (node:internal/timers:476:21)

  62) Sample human measurements:
       Sample pattern for adult cisFemale size range::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:116:16)
      at process.processImmediate (node:internal/timers:476:21)

  63) Sample human measurements:
       Sample pattern for adult cisMale size range::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:116:16)
      at process.processImmediate (node:internal/timers:476:21)

  64) Sample doll measurements:
       Sample pattern for doll cisFemale size range::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:139:20)
      at process.processImmediate (node:internal/timers:476:21)

  65) Sample doll measurements:
       Sample pattern for doll cisMale size range::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:139:20)
      at process.processImmediate (node:internal/timers:476:21)

  66) Sample giant measurements:
       Sample pattern for giant cisFemale size range::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:139:20)
      at process.processImmediate (node:internal/timers:476:21)

  67) Sample giant measurements:
       Sample pattern for giant cisMale size range::

      AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at Context.<anonymous> (file:///home/joost/git/freesewing/tests/designs/sampling.mjs:139:20)
      at process.processImmediate (node:internal/timers:476:21)


```

--- a/designs/jane/i18n/en.json
+++ b/designs/jane/i18n/en.json
@@ -32,7 +32,7 @@
       "t": "Gusset Size",
       "d": "Controls the size of the gusset"
     },
-    "bicepEase": {
+    "bicepsEase": {
       "t": "Bicep Ease",
       "d": "Controls the ease in the sleeve"
     },

--- a/designs/jane/i18n/en.json
+++ b/designs/jane/i18n/en.json
@@ -1,5 +1,5 @@
 {
-  "t": "Jane",
+  "t": "Jane 1790's shift",
   "d": "A FreeSewing pattern for a 1790s shift",
   "p": {
     "body": "Body", 

--- a/designs/jane/src/body.mjs
+++ b/designs/jane/src/body.mjs
@@ -31,6 +31,7 @@ export const body = {
     macro,
     measurements,
     part,
+    store,
   }) {
     //body
     const lengthBody =

--- a/designs/jane/src/gusset.mjs
+++ b/designs/jane/src/gusset.mjs
@@ -16,6 +16,7 @@ export const gusset = {
     macro,
     measurements,
     part,
+    store,
   }) {
     const gussetMeasure = measurements.biceps * (0.5 + 0.85) * options.gussetSize
 

--- a/designs/jane/src/sleeve.mjs
+++ b/designs/jane/src/sleeve.mjs
@@ -19,6 +19,7 @@ export const sleeve = {
     macro,
     measurements,
     part,
+    store,
   }) {
     const sleeveWidth = measurements.biceps * (1 + options.bicepsEase)
 

